tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -227,6 +248,12 @@ After(async function() {
     await Promise.all([...createdUsers.map(deleteUser), ...createdGroups.map(deleteGroup)])
   }
 
+  if (client.globals.ocis) {
+    const dataDir = user => join(client.globals.ocis_data_dir, 'data', user)
+    const deleteUserDirectories = createdUsers.map(user => fs.remove(dataDir(user)))
+    await Promise.all([...deleteUserDirectories])
```

Comment:
You can also remove https://github.com/owncloud/phoenix/pull/3865/files#diff-cdddbda94fd7c68ac41e2eaf6c69646eR233 line.
Since we have an option for running ocis and with ldap separate now, this should not be associated with ldap now.